middleware/afc_status.py

```
@@ -0,0 +1,209 @@
+
+    def _poll_loop(self) -> None:
+        """Background loop that polls AFC status at regular intervals."""
+        consecutive_failures: int = 0
+
+        while not self._stop_event.is_set():
+            data = _fetch_afc_status()
+
+            if data is not None:
+                try:
+                    _sync_lane_state(data)
+                except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)